configserver/src/main/java/com/yahoo/vespa/config/server/rpc/GetConfigProcessor.java

```diff
@@ -112,7 +112,7 @@ public void run() {
         } catch (UnknownConfigIdException e) {
             handleError(request, ErrorCode.ILLEGAL_CONFIGID, "Illegal config id " + request.getConfigKey().getConfigId());
             return;
-        } catch (Exception e) {
+        } catch (Exception | Error e) {
```

Comment:
I think you can just catch Throwable instead. In the (not very likely) event that a non-error throwable is thrown, the thread will die silently.

Reply:
fixed